lib/helpers/isBsonType.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
-const get = require('./get');
-
 /*!
  * Get the bson type, if it exists
  */
 
 function isBsonType(obj, typename) {
-  return get(obj, '_bsontype', void 0) === typename;
```

Comment:
Do we need the `typeof obj` check here?
How about `return (obj && obj._bsontype) === typename;`?

Reply:
Yes. If it was not an object, we would get an error. It is safer this way.

Reply:
Can you give me an example of a value that throws an error with the suggested solution but not with the current one?

Reply:
@AbdelrahmanHafez  If I remove the null-check we get 36 breaking unit tests.

Reply:
That shouldn't be the case, since the `&&` in `return (obj && obj._bsontype) === typename` does the `null` check. I tested locally and things are working for me. 🤨 

Anyway, either solution works well. Thanks for the performance improvement. 👍